site/content/en/docs/manual/basics/attach-cloud-storage.md

```diff
@@ -215,6 +274,28 @@ button on the `Cloud storages` page and fill out the following form:
           - `Account name` - storage account name.
 
     </br>
+
+  - [Google Cloud](https://cloud.google.com/docs):
+
+    - [`Bucket name`](https://console.cloud.google.com/storage/browser/) - cloud storage bucket name.
+
+    - `Authorization type`:
+
+      - [`Key file`](#create-a-service-account-and-key-file) - you can drag a key file to the area `attach a file`
+        or click on the area to select the key file through the explorer.
```

Comment:
how to specify the `GOOGLE_APPLICATION_CREDENTIALS` environment variable, using UI? Or should the CVAT server be set "GOOGLE_APPLICATION_CREDENTIALS" and it will be used automatically if the key file not attached?

Reply:
User should specify `GOOGLE_APPLICATION_CREDENTIALS` in the environment where CVAT instance is deployed.